int32 dmp_id
time start_time
time end_time
